content/docs/use-cases/dvc.md

```diff
@@ -14,7 +14,10 @@ $ git checkout 1-dvc-mlem-init
 ```
 
 Next let's create an isolated virtual environment to cleanly install all the
-requirements (including MLEM) there:
+requirements (including MLEM) there.
+
+This will also install DVC with `pip`. Learn about other ways to install DVC
+[here](https://dvc.org/doc/install).
```

Comment:
```suggestion
requirements with `pip` (including DVC and MLEM).

> Learn about other ways to install DVC [here](https://dvc.org/doc/install).
```

Reply:
Why do we want to mention that there are other ways to install DVC? Seems irrelevant for this use case.

Reply:
Agreed. Let's delete that mention then.